src/index.js

```diff
@@ -132,7 +133,7 @@ function handleControl(event, context) {
             }
 
             // send the change request to Lennox, send a response to Alexa on promise fulfillment
-            iComfort.setThermostatInfo(getThermostatInfoListParams, newParams.toSet, auth)
+            iComfort.setThermostatInfo(getThermostatInfoListParams, newParams.toSet)
```

Comment:
@danielmcq I believe this line should be `iComfort.setThermostatInfo(newParams.toSet)` for compatibility with the node module, if you want to adjust?